packs/data/equipment.db/grasp-of-droskar.json

```diff
@@ -0,0 +1,144 @@
+{
+    "_id": "wWXnr3xw5TzDL1M7",
+    "img": "systems/pf2e/icons/default-icons/weapon.svg",
+    "name": "Grasp of Droskar",
+    "system": {
+        "MAP": {
+            "value": ""
+        },
+        "baseItem": "gauntlet",
+        "bonus": {
+            "value": 0
+        },
+        "bonusDamage": {
+            "value": 0
+        },
+        "category": "simple",
+        "containerId": null,
+        "damage": {
+            "damageType": "bludgeoning",
+            "dice": 1,
+            "die": "d4",
+            "value": ""
+        },
+        "description": {
+            "value": "<p>This <em>+1 striking @UUID[Compendium.pf2e.equipment-srd.Ix2vicchE79d6Cl3]{Gauntlet} </em>appears little more than a dull, soot-stained, metal-plated glove at first glance, but in truth it is an invasive boon granted by Droskar to his most faithful subjects. Placing the glove on your hand causes excruciating pain as your appendage curls into a tight fist and then slowly transforms to a supernaturally hard ball of black stone, fusing to you and preventing you from using this hand for anything other than gauntlet Strikes or the<em> grasp of Droska</em>r's activated abilities (a grasp of Droskar does not have the free-hand weapon trait). The gauntlet can't be removed without a successful casting of remove curse or a similar spell. The gauntlet reverts to normal (and can be removed with ease) if the curse is removed, the hand is removed from the body, or the wearer dies.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">1</span> envision</p>\n<p><strong>Frequency</strong> once per hour</p>\n<p><strong>Effect</strong> You gain a +1 item bonus on all Athletics checks for 1 minute.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.no7vnIiNBwWjh3w8]{Effect: Grasp of Droskar}</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">2</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Effect</strong> You cast @UUID[Compendium.pf2e.spells-srd.J7Y7tl0bbdz7TcCc]{Ray of Enfeeblement} with a spell attack modifier of +10 ([[/r {1d20+10}]]{Spell Attack}) and a DC of 20.</p>"
```

Comment:
emphasize the spell links and add and inline check at the end `(@ Check[type:fortitude|dc:20|traits:necromancy,attack])`, note the space because of the dumb github bug.